_sub/compute/eks-addons/dependencies.tf

```diff
@@ -32,13 +32,13 @@ locals {
     "1.21" = "v1.21.2-eksbuild.2"
     "1.22" = "v1.22.11-eksbuild.2"
     "1.23" = "v1.23.8-eksbuild.2"
-    "1.24" = "v1.23.15-eksbuild.1"
+    "1.24" = "v1.24.9-eksbuild.1"
```

Comment:
v1.24.7-eksbuild.2 is the recommended version for kube-proxy when using managed add-ons. Reference: https://docs.aws.amazon.com/eks/latest/userguide/managing-kube-proxy.html